docs/program/apis/_index.md

```diff
@@ -204,6 +204,7 @@ The arm component supports the following methods:
 | [JointPositions](/components/arm/#jointpositions) | Get the current position of each joint on the arm. |
 | [Stop](/components/arm/#stop) | Stop the arm from moving. |
 | [IsMoving](/components/arm/#ismoving) | Get if the arm is currently moving. |
+| [Kinematics](/components/arm/#kinematics) | Get the kinematics information associated with the arm. |
```

Comment:
weren't we going to make includes so that we wouldn't have to always edit in two place @sguequierre

Reply:
@npentrel Oh I don't remember this but that seems like a good idea! do you want me to look into that in this ticket instead?

Reply:
No I think we should do that as a separate ticket since this one does mainly work on the service docs. Do you mind making a ticket?

Reply:
No problem!